ala-auth/grails-app/services/au/org/ala/web/UserListService.groovy

```diff
@@ -33,8 +33,8 @@ class UserListService {
     }
 
     private void checkConfig() {
-        if (!grailsApplication.config.getProperty('userDetails.url')) {
```

Comment:
Can we have these fall back to the old `userDetails.url` if the new property isn't set?

Reply:
sure

Reply:
done